### PR TITLE
prometheus: various fixes

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: nixos/roles/prometheus/default-alerts.nix [hunk 1]
```diff
@@ -115,18 +115,18 @@
       };
 
       TelegrafDown = {
-        expr = ''min(up{job=~"telegraf",type!='mobile'}) by (source, job, instance, org) == 0'';
```
Comment (Mic92) [inline nixos/roles/prometheus/default-alerts.nix]: Isn't this mobile filter a quite generic thing? I am using this for laptop monitoring or in general devices that are allowed to go offline.

File: nixos/roles/prometheus/default-alerts.nix [hunk 2]
```diff
@@ -75,7 +75,7 @@
       };
 
       SwapUsing30Percent = {
-        expr = ''mem_swap_total{host!="eva"} - (mem_swap_cached + mem_swap_free) > mem_swap_total * 0.3'';
```
Comment (Mic92) [inline nixos/roles/prometheus/default-alerts.nix:78]: Fine with me. This host in question doesn't even swap anymore.

**Review by Mic92 — APPROVED** [review]

File: nixos/roles/prometheus/default-alerts.nix [hunk 3]
```diff
@@ -115,18 +115,18 @@
       };
 
       TelegrafDown = {
-        expr = "min(up{job=~\"telegraf\",type!='mobile'}) by (source, job, instance, org) == 0";
```
Comment (Mic92) [inline nixos/roles/prometheus/default-alerts.nix:118]: Isn't this mobile filter a quite generic thing? I am using this for laptop monitoring or in general devices that are allowed to go offline.

Comment (zowoq) [reply nixos/roles/prometheus/default-alerts.nix:118]: I've reverted this but I don't think they belong in this repo.

**Pull request merged**